src/config.example.ts

```diff
@@ -1,5 +1,6 @@
+ 
 export default {
 	secretKey: '<Your Secret Here>',
 	mongoUrl: '<Mongo URL>',
 	PORT: 4000
-};
```

Comment:
Please don't edit unnecessary files

Reply:
Ya, i didn't changed it i was just seeing the file and I may have pressed enter that's all.